Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,45 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "without-test", "Skip running build-time tests (not recommended)"
+  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+
+  depends_on "robodoc" => :recommended
+  depends_on "cmake" => :build
+  depends_on :fortran
+
+  def install
+    mkdir "build" do
+      args = ["-DUSE_GNU_INSTALL_CONVENTION:BOOL=TRUE"] # Use more GNU/Homebrew-like install layout
+      args << "-DENABLE_UNICODE:BOOL=TRUE" if build.with? "unicode-support"
+      args << "-DSKIP_DOC_GEN:BOOL=TRUE" if build.without? "robodoc"
+      system "cmake", buildpath, *(args + std_cmake_args)
```

Comment:
@DomT4 You meant to have two periods in your first comment right? I’ll fix that to match your suggestion.

@DomT4 Also do you want me to remove the comment about `-DUSE_GNU_INSTALL_CONVENTION` or leave it?